src/main/java/com/treasure/hunt/jts/geom/GeometryAngle.java

```diff
@@ -63,6 +64,17 @@ public GeometryAngle(GeometryFactory factory, Coordinate center, double start, d
         );
     }
 
+    /** the Angle between the middle of the Hint and the x axis normalized to the range of [0 , 2*PI]
+     *
+     * @return the angle in radians
+     */
+    public double getNormalizedAngle(){
+        double angle = Angle.angle(this.getCenter() , JTSUtils.middleOfGeometryAngle(this));
+        if( angle < 0){
```

Comment:
Please use autoformatting, which prevents **many** merge conflicts in later merges.
This could be used by pressing Strg+Alt+l for one file.
Better tick all these: 
<img width="679" alt="Bildschirmfoto 2020-03-29 um 18 13 44" src="https://user-images.githubusercontent.com/14947840/77854215-2cbb1c00-71e9-11ea-9994-1456c5e8e736.png">

Reply:
Formatted